docs/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <title>Document</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <link rel="icon" href="_media/favicon.ico" />
```

Comment:
is this to quiet that console error that was always there? if so, thank you!

Reply:
This is actually to show a favicon in the docs.  I wasn't aware of the console error, but hopefully it fixes it lol

Reply:
ahh i see. haha that would be cool. maybe i'll look into the other thing.